app/js/settings.js

```diff
@@ -202,17 +205,28 @@ const init = () => {
 
 const checkOption = (setting, value) => {
     const optionList = JSON.parse(JSON.stringify(validOptions[setting]))
-    if (optionList) {
-        const valid = optionList.includes(value)
+    if (!optionList) {
+        return false
+    }
+
+    let settingValues = [value]
+    if (canAcceptMultipleOptions.has(setting)) {
+        settingValues = value.split(",")
+    }
+
+    for (const settingValue of settingValues) {
+        const valid = optionList.includes(settingValue.trim())
         if (!valid) {
             const lastOption = optionList.pop()
             const text = `'${optionList.join("', '")}' or '${lastOption}'`
             UTIL.notify(`The value of setting '${setting}' can only be one of:`
```

Comment:
If were going to implement this similar to "favoritepages", it also gives us the option to make this warning more specific and detailed, instead of showing this generic warning.